{
    "attachments": [
        {
            "name": "Screenshot",
            "source": "e3857459-a8ef-4a39-b07c-51b0c9fbc365-attachment.png",
            "type": "image/png"
        }
    ],
    "fullName": "Protractor sync test cases should fail if an element is not supposed to exist but is found #integrationSuite",
    "historyId": "Protractor sync test cases should fail if an element is not supposed to exist but is found #integrationSuite",
    "labels": [
        {
            "name": "parentSuite",
            "value": "Protractor sync test cases"
        },
        {
            "name": "host",
            "value": "Lambda"
        },
        {
            "name": "thread",
            "value": "ip-10-128-116-237"
        }
    ],
    "links": [
        {
            "name": "Test Video",
            "type": "video/mp4",
            "url": "https://allure-reports.dev.bbpd.io/test-runner/artifacts/1/18-08-27-18-10-015143/1-18-08-27-18-10-015143.mp4"
        },
        {
            "name": "Chrome Log",
            "type": "text/plain",
            "url": "https://allure-reports.dev.bbpd.io/test-runner/artifacts/1/18-08-27-18-10-015143/1-18-08-27-18-10-015143.log"
        },
        {
            "name": "Chrome Driver Log",
            "type": "text/plain",
            "url": "https://allure-reports.dev.bbpd.io/test-runner/artifacts/1/18-08-27-18-10-015143/1-18-08-27-18-10-015143.chromedriver.log"
        },
        {
            "name": "Console Log",
            "type": "text/plain",
            "url": "https://allure-reports.dev.bbpd.io/test-runner/artifacts/1/18-08-27-18-10-015143/1-18-08-27-18-10-015143.console.log"
        },
        {
            "name": "Lambda Cloudwatch Logs",
            "type": "text/html",
            "url": "https://console.aws.amazon.com/cloudwatch/home?region=us-east-1#logEventViewer:group=/aws/lambda/straferduty-manual-RunTask-1AR8QVPEGAYA8;filter=\"Test Start Locator: 3e962439-121a-4380-bda1-7efd0952c238\""
        }
    ],
    "name": "should fail if an element is not supposed to exist but is found #integrationSuite",
    "parameters": [],
    "stage": "finished",
    "start": 1535407883463,
    "status": "failed",
    "statusDetails": {
        "message": "Expected 'Error: .element-does-exist was found when it should not exist!\n    at onTimeout (/tmp/lambda_protractor/app/selection.ts:51:13)\n    at Object.polledWait (/tmp/lambda_protractor/app/polled-wait.ts:35:9)\n    at _getElements (/tmp/lambda_protractor/app/selection.ts:76:10)\n    at Object.assertElementDoesNotExist (/tmp/lambda_protractor/app/selection.ts:146:20)\n    at ElementFinderSync.assertElementDoesNotExist (/tmp/lambda_protractor/app/element-finder-sync.ts:38:12)\n    at /tmp/lambda_protractor/test/spec/protractor_sync_test.ts:70:18\n    at /tmp/lambda_protractor/test/spec/protractor_sync_test.ts:43:13\n    at fiberContents (/tmp/lambda_protractor/node_modules/asyncblock/lib/asyncblock.js:92:26)\n=== Pre-asyncblock stack ===\nError\n    at module.exports (/tmp/lambda_protractor/node_modules/asyncblock/lib/asyncblock.js:29:15)\n    at UserContext.<anonymous> (/tmp/lambda_protractor/test/spec/protractor_sync_test.ts:42:9)\n    at /tmp/lambda_protractor/node_modules/jasminewd2/index.js:108:15\n    at new ManagedPromise (/tmp/lambda_protractor/node_modules/selenium-webdriver/lib/promise.js:1077:7)\n    at ControlFlow.promise (/tmp/lambda_protractor/node_modules/selenium-webdriver/lib/promise.js:2505:12)\n    at schedulerExecute (/tmp/lambda_protractor/node_modules/jasminewd2/index.js:95:18)\n    at TaskQueue.execute_ (/tmp/lambda_protractor/node_modules/selenium-webdriver/lib/promise.js:3084:14)\n    at TaskQueue.executeNext_ (/tmp/lambda_protractor/node_modules/selenium-webdriver/lib/promise.js:3067:27)\n    at asyncRun (/tmp/lambda_protractor/node_modules/selenium-webdriver/lib/promise.js:2974:25)\n    at /tmp/lambda_protractor/node_modules/selenium-webdriver/lib/promise.js:668:7' to be undefined.",
        "trace": "Error: Failed expectation\n    at FiberFlow.errorCallback (/tmp/lambda_protractor/test/spec/protractor_sync_test.ts:48:62)\n    at FiberFlow.Flow._errorHandler (/tmp/lambda_protractor/node_modules/asyncblock-generators/lib/flow.js:530:18)\n    at fiberContents (/tmp/lambda_protractor/node_modules/asyncblock/lib/asyncblock.js:98:18)"
    },
    "steps": [],
    "stop": 1535407884181,
    "uuid": "d868d95c-9e8f-4ee6-a1dd-7c520a85f644"
}